### PR TITLE
Include version when storing study in user document

### PR DESCRIPTION
**Pull request by thomik-corp**
Provide extension version to web platform
Fixes: https://github.com/mozilla-rally/rally/issues/203

### CODE REVIEW

File: extensions/attention-stream/src/background.ts [hunk 1]
```diff
@@ -254,17 +285,30 @@ async function stateChangeCallback(newState) {
 }
 
 // Initialize the Rally SDK.
-const rally = new Rally({ enableDevMode, stateChangeCallback, rallySite, studyId, storeId, firebaseConfig, enableEmulatorMode });
+const rally = new Rally({
+  enableDevMode,
+  stateChangeCallback,
+  rallySite,
+  studyId,
+  version: browser.runtime.getManifest().version,
```
Comment (aaga) [inline extensions/attention-stream/src/background.ts:293]: I don't think this is what we want.

We don't want to pass version info _into_ the SDK, we want to be able to read the version _from_ the SDK.

Comment (aaga) [reply extensions/attention-stream/src/background.ts:293]: e.g. the version info could be exported via a public function from the Rally object? under the hood, the SDK would poll the user study doc and grab (and cache) the version from there.

**Review by aaga — CHANGES_REQUESTED** [review]

File: extensions/attention-stream/src/background.ts (hunk 1 above)
Comment (thomik-corp) [reply extensions/attention-stream/src/background.ts:293]: This is extension passing its version which is passed to the web platform. You want this to know which extension version is currently installed by the user.

Comment (thomik-corp) [reply extensions/attention-stream/src/background.ts:293]: The "latest" version is already present in the /studies database and web platform already reads that.

Comment (thomik-corp) [reply extensions/attention-stream/src/background.ts:293]: Chatted offline. The code to read user doc from SDK will be added in Ashwin's PR.

**Review by aaga — APPROVED** [review]
Stage deploy is still failing (I think due to rally-shared-types versioning) but otherwise the code looks good to me!

File: shared/types/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mozilla/rally-shared-types",
-  "version": "0.9.32",
+  "version": "0.9.33",
```
Comment (aaga) [inline shared/types/package.json:3]: @thomik-corp I think the shared-types version has already reached 0.9.34. so this should be 0.9.35?

File: web-platform/website/package.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "export-staging": "npm run build-staging && next export"
   },
   "dependencies": {
-    "@mozilla/rally-shared-types": "^0.9.32",
+    "@mozilla/rally-shared-types": "^0.9.33",
```
Comment (aaga) [inline web-platform/website/package.json:20]: see above re: version

**Pull request merged**